<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Care - 클라이언트</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="main_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_user">
      <attribute name="title">
       <string>사용자</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabUserLayout">
       <item>
        <widget class="QLabel" name="video_title">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
         <property name="text">
          <string>영상 화면</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="video_label">
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>360</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #222; color: #eee; border: 1px solid #444;</string>
         </property>
         <property name="text">
          <string>영상이 여기에 표시됩니다.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="eventHeaderLayout">
         <item>
          <widget class="QLabel" name="event_title">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>이벤트 리스트</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>9999</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_test_message">
           <property name="styleSheet">
            <string notr="true">background-color: #5a5a5a; color: white; padding: 6px 12px;</string>
           </property>
           <property name="text">
            <string>테스트 메시지 보내기</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="event_text">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>이벤트 로그가 여기에 표시됩니다.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_admin">
      <attribute name="title">
       <string>관리자</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabAdminLayout">
       <item>
        <widget class="QLabel" name="admin_label">
         <property name="text">
          <string>관리자 탭</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_vision">
      <attribute name="title">
       <string>비전 학습 모드</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabVisionLayout">
       <item>
        <widget class="QLabel" name="vision_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>비전 학습 모드 탭</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
